.github/ISSUE_TEMPLATE/component_mesheryctl.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: ⌨️ Component: mesheryctl (Meshery CLI)
+name: ⌨️ Component mesheryctl (Meshery CLI)
```

Comment:
This colon is intentional.

Reply:
@leecalcote Yes for colon to appear we need to make it `name: "⌨️ Component: mesheryctl (Meshery CLI)"` but I'm not sure it will show up as an issue template which it does not now.

Reply:
Oh, I see. Gotcha.